src/rules/util.js

```diff
@@ -87,7 +87,7 @@ export const method = node => node.parent.property;
 
 export const method2 = node => node.parent.parent.property;
 
-const methodName = node => method(node).name;
+const methodName = node => method(node) && method(node).name;
 
 const methodName2 = node => method2(node).name;
```

Comment:
Same here? `expect(something).not;` or something like that?

Reply:
I could do - I don't think it's needed per say b/c all the rules that use it check using `method` first, but I'll add it in anyway :)